src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Value.ReadTests.cs

```diff
@@ -303,6 +306,9 @@ public static void ValueFail()
             Assert.Throws<JsonException>(() => JsonSerializer.Deserialize<DateTimeOffset>(unexpectedString));
             Assert.Throws<JsonException>(() => JsonSerializer.Deserialize<DateTimeOffset?>(unexpectedString));
 
+            Assert.Throws<JsonException>(() => JsonSerializer.Deserialize<TimeSpan>(unexpectedString));
+            Assert.Throws<JsonException>(() => JsonSerializer.Deserialize<TimeSpan?>(unexpectedString));
```

Comment:
Due to other logic, we should be fine, but a few more nullable `TimeSpan` tests would be good. E.g. deserializing `null` into a non-nullable `TimeSpan` should fail, but should be fine for nullable `TimeSpan?`.